#this.css

```diff
@@ -0,0 +1,31 @@
+/*our CSS file. */
+img{
+    border-radius:10px;
+  }
+  
+#first-cointainer{
```

Comment:
check whether Image and title are placed in a same row

Reply:
I align the image with Title with commits of name --"changes"